app/config.py

```diff
@@ -20,6 +22,23 @@ def get_abs_path(file_path: str):
         return os.path.join(ROOT_DIR, file_path)
 
 
+def parse_environment(env_var: str) -> Optional[Any]:
+    """Convert environment string into Python object
+
+    Args:
+        env_var (str): environment constant, example: '["domain1.org", "domain2.com"]'
+
+    Returns:
+        Any
+    """
+    try:
+        return literal_eval(env_var)
+    except Exception as e:
+        print(
```

Comment:
I think it's better to not catch the error here so we can be informed of any failed configuration ("fail fast" principle) :)

Reply:
In this case, we can even remove `parse_environment` and just directly use `literal_eval`